clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -2067,6 +2050,14 @@ raw2trace_t::should_omit_syscall(raw2trace_thread_data_t *tdata)
 #endif
 }
 
+bool
+raw2trace_t::is_marker_type(const offline_entry_t *entry, trace_marker_type_t marker_type)
```

Comment:
I would say move the implementation to the header so it's inlined; or else explicitly mark as inline.

Reply:
Marked definition as inline.